pkg/cmd/run.go

```diff
@@ -137,14 +137,22 @@ doppler run --mount secrets.json -- cat secrets.json`,
 		dopplerSecrets := fetchSecrets(localConfig, enableCache, fallbackOpts, metadataPath, nameTransformer, dynamicSecretsTTL)
 
 		mountPath := cmd.Flag("mount").Value.String()
-		mountFormat := cmd.Flag("mount-format").Value.String()
+		mountFormatString := cmd.Flag("mount-format").Value.String()
```

Comment:
Can you separate the `Added secrets mount support for ASP.NET core JSON settings files` commit into two commits? One commit will be a `chore` to refactor the existing strings into constants, and the other will be to add support for the new format. That will make reviewing the important stuff (i.e. the new logic) really easy.

Separately, you'll want to use the imperative mood in your commits[^0]. So `Added` -> `Add`

[^0]: https://cbea.ms/git-commit/#imperative

Reply:
Done!

I also moved the `IsDotNETSettingsFile` function into its own `pkg/utils/secrets_mount.go` file as it it's related to name transformers.